erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -468,13 +478,22 @@ def apply_taxes(self):
 	def set_amounts(self):
 		self.set_received_amount()
 		self.set_amounts_in_company_currency()
-		self.set_amounts_after_tax()
```

Comment:
`set_amounts` is also called from `get_payment_entry` function. `set_amounts_after_tax ` function should also be called separately there?

Reply:
set_amounts is called from `validate`. Why call from `get_payment_entry`. Can think of any reason